src/utils/debounce.ts

```diff
@@ -0,0 +1,15 @@
+const debounce = (handler: (...arg: any) => any, timer: number) => {
+  let _timeOut: NodeJS.Timeout;
```

Comment:
why an _ ?

Reply:
Why not :-p Sometimes i use this syntax for the uninterestings let/var, but if you prefer, i clean.